HOW-TO-RUN-A-SERVER.md

```diff
@@ -1,4 +1,4 @@
-# How to run a Server
+[# How to run a Server
```

Comment:
Looks like the `[` may have been added by mistake, could you remove it?

Reply:
Whoops. Now how did that get in there!